docs/UserGuide.md

```diff
@@ -14,9 +16,9 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `tutoraid.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
```

Comment:
Should this be https://github.com/AY2122S1-CS2103T-W16-3/tp/releases ?

Reply:
Thanks for pointing this out! I didn't think of changing the link earlier since we didn't have any releases yet, but I will change it to the link you have given